experiment/test_config.yaml

```diff
@@ -1027,7 +1026,7 @@ nodeCommon:
 nodeImages:
   cosbeta:
     args:
-    - --node-args=--images=cos-stable-63-10032-71-0-p
+    - --node-args=--images=cos-beta-64-10176-60-0
```

Comment:
FYI, I'm working on https://github.com/kubernetes/test-infra/issues/3320 to avoid updating the image everytime.

Reply:
Nice!

But for now... This looks correct, right?

Or should we skip this update for now?

Thanks!
Filipe

Reply:
Yeah, this is good. Already LGTM'ed.